packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.test.tsx

```diff
@@ -39,6 +39,17 @@ describe('MessageBar', () => {
         const dismissElement = wrapper.find('.ms-MessageBar-dismissal');
         expect(dismissElement.exists()).toBe(false);
       });
+
+      it('mixes in native props to the inner text element', () => {
```

Comment:
:+1:

Reply:
Longer term we probably want to do this at the compliance test level... (@dzearing) but might be out of scope here for you just because I don't know how many other Fabric components are in need of fixing. 

(then again, it might be good to proactively go fix all Fabric components...)

Reply:
> (then again, it might be good to proactively go fix all Fabric components...)

Been thinking this as well re: adding & testing at the compliance level given how often this is asked for.